From b4eafa8b04802f45a710e712f2ec2676e0a77024 Mon Sep 17 00:00:00 2001
From: Michael Kostylev <michael.kostylev@gmail.com>
Date: Thu, 13 May 2010 10:02:04 +0000
Subject: [PATCH] Support writing 2d float arrays.

Patch by Michael Kostylev, michael D kostylev A gmail

Originally committed as revision 23105 to svn://svn.ffmpeg.org/ffmpeg/trunk
(cherry picked from commit 56b37cd1c7f3b5e5a3ece043c7ffccf70874cfb1)

Unbreaks compilation since the HE-AAC v2 backport
Signed-off-by: Reinhard Tartler <siretart@tauware.de>
---
 libavcodec/tableprint.c |    1 +
 libavcodec/tableprint.h |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libavcodec/tableprint.c b/libavcodec/tableprint.c
index e39606b..0f422eb 100644
--- a/libavcodec/tableprint.c
+++ b/libavcodec/tableprint.c
@@ -33,6 +33,7 @@ WRITE_1D_FUNC(float,  float,    "%.18e", 3)
 WRITE_2D_FUNC(int8,   int8_t)
 WRITE_2D_FUNC(uint8,  uint8_t)
 WRITE_2D_FUNC(uint32, uint32_t)
+WRITE_2D_FUNC(float,  float)
 
 void write_fileheader(void) {
     printf("/* This file was generated by libavcodec/tableprint */\n");
diff --git a/libavcodec/tableprint.h b/libavcodec/tableprint.h
index d81af97..76ab802 100644
--- a/libavcodec/tableprint.h
+++ b/libavcodec/tableprint.h
@@ -66,6 +66,7 @@ void write_float_array    (const float    *, int);
 void write_int8_2d_array  (const void *, int, int);
 void write_uint8_2d_array (const void *, int, int);
 void write_uint32_2d_array(const void *, int, int);
+void write_float_2d_array (const void *, int, int);
 /** \} */ // end of printfuncs group
 
 /** Write a standard file header */
-- 
1.7.3.5

